scripts/20230826_set-encryption-boundary-shift-true/set-encryption-boundary-shift-true.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable */
+/**
+ * This script sets encryptionBoundaryShift flag for all forms to true.
+ * A session is used to dry run the update operation before committing the changes.
+ */
+
+// Start a session.
+session = db.getMongo().startSession( { readPreference: { mode: "primary" } } );
+
+// Start a transaction
+session.startTransaction( { readConcern: { level: "local" }, writeConcern: { w: "majority" } } );
+
+formsCollection = session.getDatabase("formsg").forms;
+
+console.log('=== before ===')
+
+// total # of documents
+const totalDocs = formsCollection.countDocuments()
+console.log('total # of documents:', totalDocs);
+
+// # of documents with encryptionBoundaryShift set to true
+const docsSetToTrueBefore = formsCollection.countDocuments({ encryptionBoundaryShift: true })
+console.log('# of documents with encryptionBoundaryShift set true:', docsSetToTrueBefore);
+
+// # of documents to modify
+console.log('# of documents to modify:', totalDocs - docsSetToTrueBefore);
+
+
+// set documents
+formsCollection.updateMany(
```

Comment:
/nit: we only need to set for storage mode forms

Reply:
Updated the script to unset the field instead as we are no longer using form-level flags for the BE when this goes out! Done in [ab5d256](https://github.com/opengovsg/FormSG/pull/6665/commits/ab5d256863f0534fed04dffb60e52e8f5de23f2c)